maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/ConsoleOutputFileReporter.java

```diff
@@ -91,7 +93,7 @@ public void writeTestOutput( byte[] buf, int off, int len, boolean stdout )
                     reportsDirectory.mkdirs();
                 }
                 File file = getReportFile( reportsDirectory, reportEntryName, reportNameSuffix, "-output.txt" );
-                fileOutputStream = new FileOutputStream( file );
+                fileOutputStream = new BufferedOutputStream( new FileOutputStream( file ), 16 * 1024 );
```

Comment:
Can you please override (git amend) this commit and add Writer.close() in finally block?